substrate/frame/support/src/storage/mod.rs

```diff
@@ -1473,6 +1517,14 @@ impl<T: Encode> StorageDecodeLength for Vec<T> {}
 impl<T: Encode> StorageAppend<T> for BTreeSet<T> {}
 impl<T: Encode> StorageDecodeLength for BTreeSet<T> {}
 
+/// Blanket implementation StorageDecodeNonDedupLength
+/// for all types that are StorageDecodeLength.
```

Comment:
```suggestion
// Blanket implementation StorageDecodeNonDedupLength
// for all types that are StorageDecodeLength.
```

impl blocks need not be `///`.